assets/js/add-task.js

```diff
@@ -0,0 +1,51 @@
+import { CLICK_EVENT } from "/assets/js/constants.js";
+import showTask from "/assets/js/show-task.js";
+
+function addTaskEvent(
+  createButton,
+  cancelButton,
+  form,
+  inputValue,
+  allTasks,
+  id,
+  taskList
+) {
+  createButton.addEventListener(CLICK_EVENT, () => {
+    if (form.className == "hide") {
```

Comment:
Please use `===` for strict checking
```suggestion
    if (form.className === "hide") {
```